contribute/design/1.x/apis/cstor_volume.md

```diff
@@ -0,0 +1,320 @@
+# Move `CStorVolumes`, `CStorVolumeReplica` resources Schema to version `v1`
+
+## Table of Contents
+
+* [Introduction](#introduction)
+* [Goals](#goals)
+* [Non-Goals](#non-goals)
+* [Schema Proposal](#volume-schema-proposal)
+* [Risk and Mitigations](#risks-and-mitigations)
+* [CRD Migration and Graduation](#graduation-criteria)
+
+## Introduction
+
+This proposal highlights the limitations of current `CStorVolumes` and `CStorVolumeReplicas` schema and
```

Comment:
n_: This one doesn't speak about any limitations rather enchancements and migration to v1.

Reply:
fixed